### ISSUE TITLE
Support seeding

### ISSUE THREAD

**Issue by naggie** [jefflovejapan/drench#2]
I think this is important.

**Comment by jefflovejapan** [comment]
Yeah, totally agree, just haven't had time to commit to the project. If you're looking for a better-maintained command line client you may want to check out [bt-twisted](https://github.com/staceysern/bt-twisted) or [tulva](https://github.com/jtakkala/tulva).
